src/w3d/renderer/render2dsentence.cpp

```diff
@@ -686,11 +686,11 @@ void Render2DSentenceClass::Allocate_New_Surface(const unichar_t *text, bool reu
         text_width += m_font->Get_Char_Spacing(text[index]);
     }
 
-    int char_height = m_font->Get_Char_Height() + 1;
+    int char_height = m_font->Get_Char_Height();
```

Comment:
It would make more sense to add the 1 here as per the original rather than recompute it every loop (although yeah im sure that optimiser would do that anyway)

Reply:
Either is fine, but yeah if it is added in the loop down below, might just add 1 here instead.